packages/wallet-service/src/types.ts

```diff
@@ -745,7 +746,7 @@ export interface TxByIdToken {
   version: number;
   voided: boolean;
   weight: number;
-  balance: Balance;
+  balance: bigint;
```

Comment:
Same as the other comment about the `Balance` type.

Reply:
This is the exact same issue but with the `TxByIdToken` interface, used by `getTransactionById` method which is only used on 1 api and the result is passed to `JSON.stringify` without accessing any field.

Curiously there are other `getTransactionById` methods but these are not the same and do not use the `TxByIdToken` interface.